README.md

```diff
@@ -0,0 +1,184 @@
+[![Code Quality](https://github.com/ChrisThompsonK/team4-job-app-frontend/actions/workflows/code-quality.yml/badge.svg)](https://github.com/ChrisThompsonK/team4-job-app-frontend/actions/workflows/code-quality.yml)
+
+[![Formatted with Biome](https://img.shields.io/badge/Formatted_with-Biome-60a5fa?style=flat&logo=biome)](https://biomejs.dev/)
+
+# Team 4 Job Application Project
+
+A modern Node.js application built with TypeScript and Express.js using ES modules. This project includes modern development tooling with Biome for linting and formatting.
+
+## Features
+
+- **TypeScript** - Type-safe JavaScript development
+- **Express.js** - Fast, unopinionated web framework
+- **ES Modules** - Modern JavaScript module system
+- **Biome** - Fast linter and formatter for consistent code quality
+- **Hot Reloading** - Automatic server restart during development
+
+## Prerequisites
+
+- Node.js (v18 or higher)
+- npm
+
+## Installation
+
+1. Clone the repository:
+```bash
+git clone https://github.com/ChrisThompsonK/team4-job-app-frontend.git
+cd team4-job-app-frontend
```

Comment:
This is the backend - do we need to have a look at the readme and change it to be suitable for the backend?